mock-core-configs/etc/mock/templates/centos-stream-8.tpl

```diff
@@ -27,132 +27,128 @@ protected_packages=
 module_platform_id=platform:el8
 user_agent={{ user_agent }}
 
+{% set stream = dnf_vars['stream'] -%}
+{% set contentdir = dnf_vars['contentdir'] -%}
+
 {% if koji_primary_repo != None and koji_primary_repo != "centos-stream" %}
 [local-centos-stream]
 {% else %}
 [local]
 {% endif %}
 name=CentOS Stream $releasever - Koji Local - BUILDROOT ONLY!
-baseurl=https://koji.mbox.centos.org/kojifiles/repos/dist-c{{ releasever }}-stream-build/latest/$basearch/
+baseurl=https://koji.mbox.centos.org/kojifiles/repos/dist-c{{ stream }}-build/latest/$basearch/
```

Comment:
Please do not switch away from `$releasever-stream`

Reply:
@Conan-Kudo The goal here is to keep variables in the same places that [they appear upstream](https://git.centos.org/rpms/centos-repos/blob/11643f64f7c88715c5b0564158ff2a851b713b1e/f/SOURCES/CentOS-Stream-BaseOS.repo#_14). Is your comment specific to Koji?

Reply:
We deliberately deviate because extended variable support does not exist in yum. None of those extra variables you're adding will work on EL7.

Reply:
> We deliberately deviate because extended variable support does not exist in yum.

That's not correct. Yum supports extended variables — [even in RHEL 6](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/deployment_guide/sec-using_yum_variables). It is mock that doesn't handle them.

> None of those extra variables you're adding will work on EL7.

These are Jinja2 variables, not Yum variables, so yes they do work.